docs/source/basic.rst

```diff
@@ -1,24 +1,24 @@
 Basic executors
 ===============
 
-Mirakuru's :class:`~mirakuru.base.Executor` is something that You'll use, when you'll
+Mirakuru :class:`~mirakuru.base.Executor` is something that you will use when you
 need to make some code dependant from other process being run, and in certain state,
 and you wouldn't want this process to be running all the time.
 
-Tests would be best example here, or a script that sets up processes and databases
+Tests would be best example here or a script that sets up processes and databases
 for dev environment with one simple run.
 
 Executor
```

Comment:
As stated in the first comment of this PR I do not solve #45 I just reviewed existing documentation. Added just a few new things.